frontend/lib/src/components/widgets/AudioInput/styled-components.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import styled from "@emotion/styled"
+
+import { WAVEFORM_HEIGHT } from "./constants"
+
+export const StyledAudioInputContainerDiv = styled.div()
+
+export const StyledWaveformContainerDiv = styled.div(({ theme }) => ({
+  height: WAVEFORM_HEIGHT,
+  width: "100%",
+  background: theme.genericColors.secondaryBg,
+  borderRadius: 8,
+  marginBottom: 2,
+  display: "flex",
+  alignItems: "center",
+  position: "relative",
+  paddingLeft: 4,
+  paddingRight: 8,
+}))
+
+export const StyledWaveformInnerDiv = styled.div({
+  flex: 1,
+})
+
+export const StyledWaveSurferDiv = styled.div<{ show: boolean }>(
+  ({ show }) => ({
+    display: show ? "block" : "none",
+  })
+)
+
+export const StyledWaveformTimeCode = styled.span<{
+  isPlayingOrRecording: boolean
+}>(({ theme, isPlayingOrRecording }) => ({
+  margin: 8,
```

Comment:
**suggestion:** Should be using the existing `spacing` scale (this is true for everything in this file)